kafka/handle.go

```diff
@@ -205,3 +246,43 @@ func (h *handle) cgoGet(cgoid int) (cg cgoif, found bool) {
 
 	return cg, found
 }
+
+func (h *handle) setOAuthBearerToken(oauthBearerToken OAuthBearerToken) error {
+	cTokenValue := C.CString(oauthBearerToken.TokenValue)
+	defer C.free(unsafe.Pointer(cTokenValue))
+	cPrincipal := C.CString(oauthBearerToken.Principal)
+	defer C.free(unsafe.Pointer(cPrincipal))
+	cErrstrSize := C.size_t(512)
+	cErrstr := (*C.char)(C.malloc(cErrstrSize))
+	defer C.free(unsafe.Pointer(cErrstr))
+	cExtensions := make([]*C.char, 2*len(oauthBearerToken.Extensions))
```

Comment:
nit: I wouldn't mind some more air in this function. Blank lines between sections

Reply:
👍